vendor/licenses/git_submodule/koka-textmate-grammar.dep.yml

```
@@ -0,0 +1,139 @@
+---
+name: koka-textmate-grammar
+version: 66402e23e04f4e8b8abdf67093593644c110f0fe
+type: git_submodule
+homepage: https://github.com/koka-community/koka-textmate-grammar.git
+license: other
```

The license detection hasn't succeeded to automatically detect the license so we need to set it manually.

```suggestion
license: apache-2.0
```